### ISSUE TITLE
entities not created

### ISSUE THREAD

**Issue by frankyhun** [thecode/ha-onewire-sysbus#4]
I installed this custom integration via HACS, did the restart. In the /sys/bus/w1/devices/ the one wire devices can be seen:
28-01143f60a7aa ... etc.

But the sensor.28.<deviceId> entities are not present in HA.

**Comment by frankyhun** [comment]
Here is the diagnostics file:
[config_entry-hacs-568891adda9927ef57769072e8c970bc.json.txt](https://github.com/thecode/ha-onewire-sysbus/files/8826503/config_entry-hacs-568891adda9927ef57769072e8c970bc.json.txt)

**Comment by thecode** [comment]
> Here is the diagnostics file: [config_entry-hacs-568891adda9927ef57769072e8c970bc.json.txt](https://github.com/thecode/ha-onewire-sysbus/files/8826503/config_entry-hacs-568891adda9927ef57769072e8c970bc.json.txt)

Please download the diagnostics from the 1-wire sysbus not from HACS:

![image](https://user-images.githubusercontent.com/1858925/171701122-b9d8bc34-3039-4dc4-8dc0-c434c3808b31.png)

Did you add the integration after installing?
![image](https://user-images.githubusercontent.com/1858925/171701204-0dae274d-f3d4-4cb6-8943-17068b8ec0e4.png)

![image](https://user-images.githubusercontent.com/1858925/171701304-b93bf8c0-a970-49fd-9957-2653ea700752.png)

**Comment by owcazz** [comment]
Yes, now there is integration.

**Comment by frankyhun** [comment]
@thecode turned out, the problem was with the 1-wire sysbus integration. It was installed before i did a HA core upgrade to core-2022.6.0. After the upgrade i realised, that the Raspberry GPIOs and one wire is not part of HA Core anymore. I found this plugin, so i installed HACS, and ha-onewire-sysbus. I remembered, that 1-wire sysbus was installed before, so i was not looking at that plugin, only searched the logs for any errors.

Long story short: the 1-wire sysbus integration reported an error, i reinstalled it, and everything is working now as expected.
Thank you for your time!

**Comment by bogdancandea** [comment]
I followed this topic to fix similar problem but i can't find 1Wore SysBus integration.
![image](https://user-images.githubusercontent.com/113257587/190895186-52eb69cd-39ac-4ebd-87d6-7c1315e76e4e.png)

Thanks for any help.

**Comment by bogdancandea** [comment]
It seems that host reboot was needed to see 1-wire sysbus integration. 

Anyway, this extra information shall be added to instruction page
